app-server/app/src/main/scala/user/Routes.scala

```diff
@@ -44,10 +41,10 @@ trait UserRoutes extends Authentication {
               }
             } ~
             put {
-              entity(as[UsersRow]) {
+              entity(as[UsersRowApi]) {
```

Comment:
Shouldn't this just be `UsersRow` -- users are going to be submitting the full object back, we just ignore the read only fields